.github/workflows/issues_body-cleanup.yml

```diff
@@ -34,3 +32,5 @@ jobs:
           script: |
             const script = require('./.github/workflows/scripts/issues/bodyCleanup.js')
             await script({core, github, context})
+    outputs:
+      orderId: ${{ steps.cleanup.outputs.ORDER_ID }}
```

Comment:
I personally don't like this.
Key order is irrelevant in YAML, but all official examples and the best practices everywhere have the `steps` key last. And IMHO it makes perfect sense. 🤷🏼

Reply:
Oh ok, I thought:

1. inputs
2. steps
3. outputs

😄

Reply:
Yeah... from a process standpoint it does make sense, but when you think about it as definition (which it technically is, as it is just a mapping) it does make more sense like that.

1. definition
2. execution